graphql_api/types/bundle_analysis/report.py

```diff
@@ -3,11 +3,10 @@
 from ariadne import ObjectType, UnionType
 
```

Comment:
similar to `base.py`, added implementation for new bundleData and marked old ones to be deprecated